lib/Index/Index.cpp

```diff
@@ -913,10 +913,18 @@ class IndexSwiftASTWalker : public SourceEntityWalker {
     // report an occurrence of `foo` in `_foo` and '$foo').
     if (auto *VD = dyn_cast<VarDecl>(D)) {
       if (auto *Wrapped = VD->getOriginalWrappedProperty()) {
-        assert(Range.getByteLength() > 1 &&
-               (Range.str().front() == '_' || Range.str().front() == '$'));
-        auto AfterDollar = Loc.getAdvancedLoc(1);
-        reportRef(Wrapped, AfterDollar, Info, std::nullopt);
+        assert(Range.getByteLength() > 1);
+        if (Range.str().front() == '`') {
+          assert(Range.getByteLength() > 3);
+          assert(Range.str().consume_front("`_") ||
+                 Range.str().consume_front("`$"));
```

Comment:
I think you want `front` rather than `consume_front`. It's just mutating the temporary, so it's not going to cause a problem despite being in the `assert`.